content/v2.0/influxdb-templates/stacks/_index.md

```diff
@@ -39,52 +37,118 @@ Use stacks to save time in the following cases:
 ### Automate deployments with GitOps and stacks
 
 GitOps is popular way to configure and automate deployments. Use InfluxDB stacks in a GitOps workflow
```

Comment:
After sentence that starts on line 39:

To automate deployments with GitOps and stacks, complete the following steps:
1. [Set up a GitHub repository to back your InfluxDB instance.](#set-up...)
2. [Populate your GitHub repository with the existing resources in your instance.](#populate...)
3. [Automate the creation of a stack for each folder.](#automate-the...)
4. [Automate deployment using Github Actions or CircleCI.](#automate-dep...)